### PR TITLE
feat: Use opencollective for support

### PR DESCRIPTION
**Pull request by coder3101**
## Description
Opencollective is a Fiscal host for open source projects. The transactions and expenses are tracked by the website and all data is made available to public. It supports many features like Goal to raise funds, One time donations, etc. It is safe and widely available and transparent. 

Our page is located at : https://opencollective.com/cpeditor

## Related Issues / Pull Requests
N/A

## Motivation and Context
Be ready with funding.

## How Has This Been Tested?
Arch

## Checklist
N/A

## Additional text
N/A

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,6 +13,10 @@ It makes your competitive coding life easier :grin: by automating many things fo
 <!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
 [![All Contributors](https://img.shields.io/badge/all_contributors-48-orange.svg)](#all-contributors)
 <!-- ALL-CONTRIBUTORS-BADGE:END -->
+
+[![Open Collective](https://opencollective.com/cpeditor/tiers/backer.svg?avatarHeight=36)](https://opencollective.com/cpeditor#section-contribute)
```
Comment (ouuan) [inline README.md:17]: This badge looks different from the others, so I think it should be above the all-contributors badge.

**Comment by ouuan** [comment]
What about using "Become a backer" in the menu, too?

**Comment by coder3101** [comment]
> What about using "Become a backer" in the menu, too?

Do you mean we should rename it from "Support us" to "Become a backer"?

File: README.md (hunk 1 above)
Comment (coder3101) [reply README.md:17]: Sure!

**Comment by ouuan** [comment]
> > What about using "Become a backer" in the menu, too?
> 
> Do you mean we should rename it from "Support us" to "Become a backer"?

Yes.

**Comment by ouuan** [comment]
We can add a FUNDING.yml.

**Comment by coder3101** [comment]
> We can add a FUNDING.yml.

For GitHub Sponsors?

I tried to set it up with Fiscal host as open collective. But GitHub Sponsors isn't available in India and it needs an Organization bank account.

**Comment by ouuan** [comment]
> > > What about using "Become a backer" in the menu, too?
> > 
> > 
> > Do you mean we should rename it from "Support us" to "Become a backer"?
> 
> Yes.

I'm not very sure which is better. We may refer to other software and see how they call this action.

**Comment by coder3101** [comment]
What is left of this PR?

**Comment by coder3101** [comment]
"Donate" word is used by KDE apps like Konsole in Help menu. neovim uses "default" buffer to show some tips and also shows the "Support us" with some description as to why they need support.

I think "Support us" is fine for now, when we fix #115 we can support label with description there as well.

**Comment by coder3101** [comment]
@IZOBRETATEL777 Please update the translations.

File: README.md [hunk 2]
```diff
@@ -10,9 +10,12 @@ It makes your competitive coding life easier :grin: by automating many things fo
 
 ---
 
+[![Open Collective](https://opencollective.com/cpeditor/tiers/support-us.svg?avatarHeight=60)](https://opencollective.com/cpeditor#section-contribute)
+<br>
```
Comment (ouuan) [inline README.md:14]: Why `<br>`?

Comment (coder3101) [reply README.md:14]: It wasn't breaking a line in Preview. When I added it worked.

Comment (coder3101) [reply README.md:14]: You can preview what looks better, with or without line break.

**Comment by coder3101** [comment]
You can merge this PR after approving.

**Review by ouuan — APPROVED** [review]

**Pull request merged**